WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -51,17 +51,8 @@ extension WooTab {
     }
 
     // Note: currently only the Dashboard tab (My Store) view controller is set up in Main.storyboard.
-    private static func visibleTabs(_ isHubMenuFeatureFlagOn: Bool) -> [WooTab] {
-        var tabs: [WooTab] = [.myStore, .orders, .products]
-
-        if isHubMenuFeatureFlagOn {
-            tabs.append(.hubMenu)
-        }
-        else {
-            tabs.append(.reviews)
-        }
-
-        return tabs
+    private static func visibleTabs() -> [WooTab] {
+        return [.myStore, .orders, .products, .hubMenu]
```

Comment:
Despite we're not calculating this anymore, I decided to leave the method here so we don't repeat the same logic inside `visibleIndex()` and the `init()`, and because of the internal note. In any case, happy to refactor as needed 👍